analytics_data_api/v0/tests/test_connections.py

```diff
@@ -0,0 +1,35 @@
+from django.test import TestCase
+from django.test.utils import override_settings
+from elasticsearch.exceptions import ElasticsearchException
+from mock import patch
+
+from analytics_data_api.v0.connections import BotoHttpConnection, ESConnection
+
+
+class ESConnectionTests(TestCase):
+
+    @override_settings(ELASTICSEARCH_CONNECTION_DEFAULT_REGION='region_123')
+    def test_region(self):
+        connection = ESConnection('mockservice.cc-zone-1.amazonaws.com',
+                                  aws_access_key_id='access_key',
+                                  aws_secret_access_key='secret')
+        self.assertEqual(connection.auth_region_name, 'region_123')
+
+
+@override_settings(ELASTICSEARCH_AWS_ACCESS_KEY_ID='access_key')
+@override_settings(ELASTICSEARCH_AWS_SECRET_ACCESS_KEY='secret')
+class BotoHttpConnectionTests(TestCase):
+
+    @patch('analytics_data_api.v0.connections.ESConnection.make_request')
+    def test_perform_request_success(self, mock_response):
+        mock_response.return_value.status = 200
+        connection = BotoHttpConnection()
+        status, _header, _data = connection.perform_request('get', 'http://example.com')
+        self.assertEqual(status, 200)
```

Comment:
The `elasticsearch-py` tests don't run in our builds, let alone with our `BotoHttpConnection` connection class, though, so if we don't write our own test, that functionality will never be checked.